### ISSUE TITLE
[SUGGESTION] Ability to choose ship type with different control schemes

### ISSUE THREAD

**Issue by matejkar** [tsunamayo/Starship-EVO#3090]
The current flying system in the game is quite good, but im not a bit fan of the current system of the free look mode using the TAB key, making the cursor stuck at the center of the ship.
Now i have a suggestion to add the ability to choose a ship type in the starship computer menu, with the choices being capital ship and spacecraft. The capital ship would have a control scheme like in the games like dreadnought or fractured space:
 capitals: space - elevate, shift - descend, wasd - forward/turn left/reverse/turn right, q and e - roll, and f and g for drifting.
 The most notable change would be in the camera movement, being switched to the camera mode of the dreadnought game, where the cursor is, the player is looking there, and can be locked by pressing and holding RMB. I hope this makes larger ships more comfortable to control as the controls can be a bit wanky (for example, shooting a target out of sight, and no ability to keep turning towards them and free look at their position). Also capitals would not be able to dock

now for smaller ships, i would leave the controls as they are

**Comment by WILDCATreactor** [comment]
While this kind of "tank-style" control scheme is fun and convenient, it reduces several factors of immersion and teamwork that a game like Dreadnought is not designed around.  Currently you can use the cameras and displays, linked to hotkeys, to help ease of control issues you might find in a design, so you don't necessarily need [tab]+flight control functionality when a docking cam could be designed (assuming proper docking is to be added).  The only remaining reason you might need such a control/look scheme is if you intend to fly and fight in a one-man dreadnought, a la the namesake game and many others, which I think flies in the face of survival/crafting games.

**Comment by matejkar** [comment]
@WILDCATreactor if u are talking about the ability to multi crew a ship, then the players should be able to simply change the ship control to that of a spacecraft, with the current control scheme.

**Comment by WILDCATreactor** [comment]
Yes, I'm submitting that you should be limited in needing to multi-crew very large ships to remain effective- that's kind of the primary balance factor in gameplay on them.
